schemas/context/viewability-details.schema.json

```diff
@@ -7,18 +7,17 @@
   ],
   "$id": "https://ns.adobe.com/xdm/context/viewability-details",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "Viewability",
+  "title": "Viewability Details",
   "type": "object",
-  "description":
-  "Description of viewability measured properties of an advertisement",
+  "description": "Browser or app settings measured at runtime describing the viewability of an advertisement or content associated with each page load.",
```

Comment:
This needs way more detail. Probably best to move it into a separate file.

Reply:
Added viewability-details.description.md

Reply:
GREAT description, especially the reference to the spec.  Thanks!

Reply:
I agree. Great jon, @gabebarcelos-adobe